test/spark/docker-compose.yaml

```diff
@@ -61,18 +63,40 @@ services:
       - SPARK_RPC_ENCRYPTION_ENABLED=no
       - SPARK_LOCAL_STORAGE_ENCRYPTION_ENABLED=no
       - SPARK_SSL_ENABLED=no
+      - AWS_ACCESS_KEY_ID
+      - AWS_SECRET_ACCESS_KEY
+      - AWS_REGION
     env_file:
         - tester.env
     extra_hosts:
       - "s3.docker.lakefs.io:10.5.0.55"
       - "example.s3.docker.lakefs.io:10.5.0.55"
+      - "gateway-test.s3.docker.lakefs.io:10.5.0.55"
+      - "thick-client-test.s3.docker.lakefs.io:10.5.0.55"
   spark-worker-2:
     <<: *spark_worker
   spark-submit:
-    <<: *spark_worker
+    image: docker.io/bitnami/spark:3
     profiles: ["command"]
     volumes:
       - ./:/local
+    env_file:
+      - tester.env
+    environment:
+      - input
+      - output
+      - s3input
+      - s3output
```

Comment:
if possible I prefer not to pass these and env - it should of the command we execute part of the command we run.